app/views/devices/_table.html.erb

```diff
@@ -17,35 +17,13 @@
           <% end %>
         </tr>
       </thead>
-      <tbody class="bg-white divide-y dark:divide-gray-700 dark:bg-gray-800">
-        <% devices.each do |device| %>
-          <tr class="text-gray-700 dark:text-gray-400" for="<%= device.id %>">
-              <% if action_name == "import" %>
-                  <td class="px-4 py-3"><%= check_box_tag "device_ids[]", device.id, false, id: "device_ids_#{device.id}" %></td>
-              <% end %>
-              <td class="px-4 py-3"><%= device.name %></td>
-              <td class="px-4 py-3"><%= device.code %></td>
-              <td class="px-4 py-3"><%= Device.human_enum_name(:status, "#{device.status}") %></td>
-              <td class="px-4 py-3"><%= device.brand&.name %></td>
-              <td class="px-4 py-3"><%= device.category&.name %></td>
-              <td class="px-4 py-3"><%= convert_date_ymd(device.input_date) %></td>
-              <% unless action_name == "import"%>
-              <td class="px-4 py-3">
-                <%= link_to device_path(device), class: "hover:text-indigo-600" do %>
-                  <i class="fa-regular fa-eye"></i>
-                  <%= t "shared.detail" %>
-                <% end %>
-              </td>
-              <% end %>
-          </tr>
-        <% end %>
-          <tr class="text-gray-700">
-            <% if @pagy.present? %>
-              <td class="px-4 py-3 text-sm text-right" colspan="7"><%= pagy_nav(@pagy).html_safe %></td>
-            <% end %>
-          </tr>
+      <tbody class="bg-white divide-y" id="devices">
+        <tr id="devices_skeleton">
+          <%= render "devices_skeleton" %>
+        </tr>
       </tbody>
     </table>
+    <%= turbo_frame_tag :devices_pagination, src: devices_path(page: @pagy.next, format: :turbo_stream) if @pagy.next.present? %>
```

Comment:
Đây cũng vậy nè, k nên dùng Instance variable trong partial view nha

Reply:
Never create an instance variable just to share it between partials
Using @instance_variables in partials is bad design